test/pummel/test-keep-alive.js

```diff
@@ -45,7 +45,7 @@ let keepAliveReqSec = 0;
 let normalReqSec = 0;
 
 
-function runAb(opts, callback) {
+let runAb = (opts, callback) => {
```

Comment:
Nit: This can be a `const` as well.

Reply:
@thefourtheye  Sure. I understand.

Reply:
@sreepurnajasti - while you are at it, please review the linter failure and address that as well: 

https://travis-ci.com/nodejs/node/jobs/159491565#L448
https://travis-ci.com/nodejs/node/jobs/159491565#L449

thanks!

Reply:
@gireeshpunathil Oh, my bad.  I forgot about it. I will take care. Thank you.